LambdaSchool Site/index.html

```diff
@@ -0,0 +1,87 @@
+<!doctype html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+
+<meta name="description" content="I’m a freelance designer and front-end developer with a passion for details and clean interfaces."/>
+<title>Lambda School - A computer science education that's 100% free until you're hired</title>
+
+<link href="style.css" rel="stylesheet" type="text/css">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
+</head>
+
+
+ <title>The Computer Science Education of the Future</title>
+
+<body>
+   	<div class="main-container">
+      <div class="background-image" style="background: url(https://lambdaschool.com/assets/img/background.jpg); background-size: 100%; background-repeat: no-repeat;">
+		      <div class="container-nav" style="border-bottom: 2px solid; margin-bottom: 1%; color: lightgray; opacity: 0.7;">
+			         <div class="nav-bar">
+					        <ul class="nav-list">
+						          <li class="nav-logo"><img src="https://lambdaschool.com//assets/img/lambdawhite.png" style="width:100px;"></li>
+						          <li class="list-item">About</li>
+						          <li class="list-item dropdown">
+                      <a class="dropbtn" style="text-decoration: none;">Courses</a>
```

Comment:
Maybe it's just Wrapping the text here, but it looks like things are out of alignment.